amqtt/utils.py

```diff
@@ -5,12 +5,17 @@
 import random
 
 import yaml
+import typing
 
+if typing.TYPE_CHECKING:
+    from amqtt.session import Session
 
 logger = logging.getLogger(__name__)
 
 
-def format_client_message(session=None, address=None, port=None):
+def format_client_message(
+    session: "Session" = None, address: str = None, port: int = None
```

Comment:
personal preference: Using `from __future__ import annotations` instead of directly using string

Reply:
Yeah, right! A little less mixture of styles. Can you please have a second look? If you approve, feel free to merge.

Reply:
It that `python3.7+` only? Shell we deprecate `python3.6` support?